src/txn/txn_rollback_to_stable.c

```diff
@@ -527,7 +526,10 @@ __rollback_row_ondisk_fixup_key(WT_SESSION_IMPL *session, WT_PAGE *page, WT_ROW
             __wt_verbose(session, WT_VERB_RECOVERY_RTS(session), "%p: key removed", (void *)key);
         }
 
-        WT_ERR(__rollback_row_add_update(session, page, rip, upd));
+        if (rip != NULL)
+            WT_ERR(__rollback_row_add_update(session, page, rip, upd));
```

Comment:
Why the naming is different? Can we change this to __rollback_roll_modify?

Reply:
Fixed.